docs/admin/search.md

```diff
@@ -4,22 +4,21 @@ title: Search
 ---
 
 **The built in search system is used for:**
+
 - Directory
 - People/Space Search
 - Content Search
 - User/Space Picker Widgets
 
```

Comment:
This part is a bit confusing. Isn't Directory = People/Space? What's User/Space Picker Widgets?

Reply:
We had a "Directory" previously, which combined People & Spaces in one view. Let's go with:

- Content 
- People
- Spaces
- Marketplace
- Third-party resources ("Advanced Search" - HumHub Professional Edition module) (insert linke to module & PE)